micrometer-core/src/main/java/io/micrometer/core/instrument/binder/cache/EhCache2Metrics.java

```diff
@@ -118,22 +118,26 @@ protected void bindImplementationSpecificMetrics(MeterRegistry registry) {
                 .tags(getTagsWithCacheName()).tags("result", "updated").description(DESCRIPTION_CACHE_PUTS_ADDED)
                 .register(registry);
 
+        FunctionCounter.builder("cache.expired.count", stats, StatisticsGateway::cacheExpiredCount)
+                .tags(getTagsWithCacheName()).description("Cache expired count").register(registry);
+
+        Gauge.builder("cache.hitRatio", stats, StatisticsGateway::cacheHitRatio).tags(getTagsWithCacheName())
```

Comment:
We usually avoid registering metrics that you can derive from other metrics. Can't you get the hit ratio from the standard hit count and miss count metrics?